### DISCUSSION TITLE
@REMOVES@ was added to the documentation but not released yet

### DISCUSSION THREAD

**Discussion by aklimkovich** [pactumjs/pactum#173] [Q&A] [answered]
Couple days ago a new functionality for managing data templates was added : @REMOVES@
When do you plan to add it to the release? It's very useful option and I would like to use\test it. 
I'm using 3.1.9 latest now
Thanks!

**Comment by ASaiAnudeep** [comment, marked as answer]
It's released - https://github.com/pactumjs/pactum/releases/tag/v3.1.10

Thanks a lot for starting this discussion. Please do use and test it & share the feedback with us.

**Reply by aklimkovich** [reply]
@ASaiAnudeep  Thanks a lot!
